src/e2e/puppeteer/__tests__/url.ts

```diff
@@ -50,7 +50,7 @@ describe('multiline', () => {
 
     await paste(`
     - https://test.com/single-line
-    - https://test.com/some/very/very/very/very/very/very/very/very/very/very/very/very/very/very/long/url
+    - https://test.com/some/very/very/very/very/very/very/very/very/very/very/very/very/very/very/long/url/without-cursor
```

Comment:
Needed to make this url longer because it was longer ellipsizing as the test expected.